### PR TITLE
Allow kvm or zvm for sle-micro when it's s390

### PR DESCRIPTION
**Pull request by Zaoliang**
see https://progress.opensuse.org/issues/133541

### CODE REVIEW

File: tests/jeos/firstrun.pm [hunk 1]
```diff
@@ -67,7 +67,8 @@ sub verify_hypervisor {
 
     return 0 if (
         is_qemu && $virt =~ /(qemu|kvm)/ ||
-        is_s390x && $virt =~ /zvm/ ||
+        is_s390x && $virt =~ /(zvm|kvm)/ ||
+        &virt =~ /kvm/ ||
```
Comment (foursixnine) [inline tests/jeos/firstrun.pm:71]:
```suggestion
```

**Review by foursixnine — APPROVED** [review]

File: tests/jeos/firstrun.pm (hunk 1 above)
Comment (Zaoliang) [reply tests/jeos/firstrun.pm]: yes, this is covered now.

**Review by rfan1 — APPROVED** [review]

**Comment by foursixnine** [comment]
@Zaoliang I think you can remove the WIP from the tittle to merge :)

**Comment by Zaoliang** [comment]
PR looks good now with VR: 
https://openqa.suse.de/tests/11740309

**Pull request merged**